data_model/src/isi.rs

```diff
@@ -997,6 +1000,20 @@ mod transparent {
             pub msg: String,
         }
     }
+
+    isi! {
+        /// Custom instruction with arbitrary payload.
+        /// Should be handled in custom executor, where it will be translated to usual ISIs.
+        /// Can be used to extend instruction set or add expression system.
+        /// See `executor_custom_isi_simple` and `executor_custom_isi_complex`
+        /// examples in `client/tests/integration/smartcontracts`.
+        #[derive(Constructor, Display)]
```

Comment:
```suggestion
        #[derive(Display)]
```

take a look at `Permission::new` and mirror it